news_aggregator/services/operational_alerts.py

```
@@ -0,0 +1,86 @@
+            state.active = True
+            if not should_send:
+                state.suppressed += 1
+                return False
+
+            suffix = ""
+            if state.suppressed:
+                suffix = f"\n\nПовторов с прошлого уведомления: {state.suppressed}"
+            alert_message = (message + suffix)[:3200]
+            try:
+                sent = bool(await self._sender(title[:200], alert_message))
+            except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)